src/config/hlp.ts

```diff
@@ -71,3 +63,80 @@ export const HLP_CONTRACTS: NetworkMap<HlpContracts | undefined> = {
   ethereum: undefined,
   polygon: undefined
 };
+
+let loadedConfig: Record<Network, HlpConfig | undefined> = {
```

Comment:
This should be a const
```suggestion
const loadedConfig: Record<Network, HlpConfig | undefined> = {
```